PROTOCOL.md

```diff
@@ -515,11 +552,41 @@ To achieve the requirements above, related actions from different delta files ne
  - To replay the log, scan all file actions and keep only the newest reference for each logical file.
  - `add` actions in the result identify logical files currently present in the table (for queries). `remove` actions in the result identify tombstones of logical files no longer present in the table (for VACUUM).
 
+# Table Features
```

Comment:
Strong recommendation: we should remove title casing from all mentions of Table Features.

Even within this initial commit, we have 2 sections where we drop title casing and refer to "enabled features" and "disabled features", and then provide instructions relative to `readerFeatures` and `writerFeatures` rather than using the Table Feature moniker.

We should treat table features the same way we treat protocol version and remove capitalization. While this is an exciting new development for how Delta tables will be versioned today, this will immediately become a commonplace part of the Delta protocol and so not really a "feature" that should need strong emphasis.

Reply:
Agree, table features will become an essential part of Delta therefore no need to be emphasized.